schunk_gripper_driver/schunk_gripper_driver/tests/test_services.py

```diff
@@ -353,6 +353,80 @@ def test_driver_implements_loading_previous_configuration(driver):
     assert future.result().success
 
 
+@skip_without_gripper
+def test_driver_implements_grip_at_position_and_release(lifecycle_interface):
+    driver = lifecycle_interface
+
+    node = Node("check_grip_at_position")
+    add_client = node.create_client(AddGripper, "/schunk/driver/add_gripper")
+    reset_client = node.create_client(Trigger, "/schunk/driver/reset_grippers")
+    assert add_client.wait_for_service(timeout_sec=2)
+    assert reset_client.wait_for_service(timeout_sec=2)
+
+    # Reset grippers (drop default modbus gripper)
+    request = Trigger.Request()
+    future = reset_client.call_async(request)
+    rclpy.spin_until_future_complete(node, future)
+    assert future.result().success
+
+    # Add TCP/IP gripper
+    request = AddGripper.Request()
+    request.gripper.host = "0.0.0.0"
+    request.gripper.port = 8000
+    future = add_client.call_async(request)
+    rclpy.spin_until_future_complete(node, future)
+    assert future.result().success
+
+    driver.change_state(Transition.TRANSITION_CONFIGURE)
+    driver.change_state(Transition.TRANSITION_ACTIVATE)
+
+    # Get gripper services
+    for gripper in driver.list_grippers():
+        try:
```

Comment:
Right, users will have to call our driver's services like this if they want gripper-robust code. It's good to see this explicitly stated here!